### ISSUE TITLE
Entity implements deprecated device_state_attributes

### ISSUE THREAD

**Issue by masdgaf** [gazoodle/gecko-home-assistant#38]
## Version of the custom_component
v0.0.7

## Configuration

```yaml

NA

```

## Describe the bug

Entity implements deprecated device_state_atribute

## Debug log

```Entity sensor.hot_tub_smart_winter_mode_risk (<class 'custom_components.gecko.sensor.GeckoSensor'>) implements device_state_attributes. Please report it to the custom component author.
Entity climate.hot_tub_heater (<class 'custom_components.gecko.climate.GeckoClimate'>) implements device_state_attributes. Please report it to the custom component author.
Entity light.hot_tub_lights (<class 'custom_components.gecko.light.GeckoLight'>) implements device_state_attributes. Please report it to the custom component author.
Entity fan.hot_tub_pump_1 (<class 'custom_components.gecko.fan.GeckoFan'>) implements device_state_attributes. Please report it to the custom component author.
Entity fan.hot_tub_pump_2 (<class 'custom_components.gecko.fan.GeckoFan'>) implements device_state_attributes. Please report it to the custom component author.

**Comment by MrBanditt** [comment]
I don't know how to do a pull request to fix, but it seems like it is just a rename:
[https://discord.com/channels/330944238910963714/427516175237382144/916624064301391892](url)

I edited gecko-home-assistant/custom_components/gecko/entity.py
line 40
changed:
def device_state_attributes(self):
to:
def extra_state_attributes(self):

No more warnings and everything seems to be working.

**Comment by gazoodle** [comment]
Fixed in v0.0.9
